application/src/test/java/org/opentripplanner/osm/tagmapping/OsmTagMapperTest.java

```diff
@@ -199,7 +201,16 @@ void corridorName() {
     var way = way("highway", "corridor");
     assertEquals("corridor", wps.getCreativeNameForWay(way).toString());
     assertEquals("Korridor", wps.getCreativeNameForWay(way).toString(Locale.GERMANY));
-    assertEquals("käytävä", wps.getCreativeNameForWay(way).toString(Locale.of("FI")));
+    assertEquals("käytävä", wps.getCreativeNameForWay(way).toString(FI));
+  }
+
+  @Test
+  void indoorAreaName() {
+    var wps = wayProperySet();
+    var way = way("indoor", "area");
+    assertEquals("indoor area", wps.getCreativeNameForWay(way).toString());
+    assertEquals("Innenbereich", wps.getCreativeNameForWay(way).toString(Locale.GERMANY));
```

Comment:
Why did you extract a variable for Finnish but not German?

Reply:
Because the Java Locale class has only a few common locales predefined: https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/Locale.html#field-summary

Finnish is not among them.